### PR TITLE
Fabo/2228 display vesting accounts

### LINKED ISSUE #2228: Vesting account and coins support
**Is your feature request related to a problem? Please describe.**

I'd like to know if any of my coins are vesting (locked) and the time they vest so I can choose those vesting coins over the regular ones to delegate.

**Describe the solution you'd like**

- Clearly separate vesting coins from regular coins on the wallet balance
- Choose either vesting (`original_vesting`) or regular coins (`coins`) for staking (delegation, redelegation) and governance.
- If (re)delegate using my vesting coins, I want to see that the delegation was made with them and not with regular coins
- Show a message on Send action modal that the `original_vesting` coins are locked and I can't transfer them until `end_time` date
- (optional) show a time progress bar for when coins become vested (unlocked) on the Wallet Page

**Describe alternatives you've considered**

- create a separate state for vesting balances (`original_vesting`)
- add a dropdown on action modals to choose which coin to use: regular or vesting (`original_vesting`)

**[BaseVestingAccount](https://github.com/cosmos/cosmos-sdk/blob/4b765b6ab4dfac38a896b66478b3b14b40d0c731/x/auth/account.go#L178-L191
)**

A vesting account has the following additional fields:

```
type BaseVestingAccount struct {
	*BaseAccount

	OriginalVesting  sdk.Coins `json:"original_vesting"`  // coins in account upon initialization
	DelegatedFree    sdk.Coins `json:"delegated_free"`    // coins that are vested and delegated
	DelegatedVesting sdk.Coins `json:"delegated_vesting"` // coins that vesting and delegated

	EndTime int64 `json:"end_time"` // when the coins become unlocked
}
```

**[ContinuousVestingAccount](https://github.com/cosmos/cosmos-sdk/blob/4b765b6ab4dfac38a896b66478b3b14b40d0c731/x/auth/account.go#L347-L353)** 

Same as the `BaseVestingAccount`, but it additionally has a **start_time** for `when the coins start to vest`:

```
type ContinuousVestingAccount struct {
	*BaseVestingAccount

	StartTime int64 `json:"start_time"` // when the coins start to vest
}
```

AC:
- [x] display vesting accounts tokens
- [ ] display vesting account information

### PR DESCRIPTION
**Pull request by faboweb**
Part of #2228 

**Description:**

Simple version where we display the coins as normal coins. This is the lease code change (more like a bugfix). We can think about adding more vesting info later.

Thank you! 🚀

---

For contributor:

- [ ] Added entries in `PENDING.md` with issue # and GitHub username
- [ ] Reviewed `Files changed` in the github PR explorer
- [ ] Attach screenshots of the UI components on the PR description (if applicable)
- [ ] Scope of work approved for big PRs

For reviewer:

- [ ] Manually tested the changes on the UI

### CODE REVIEW

File: PENDING.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+### Fixed
```
Comment (fedekunze) [inline PENDING.md]: this should go to added

File: app/src/renderer/connectors/api.js [hunk 2]
```diff
@@ -45,7 +45,16 @@ const Client = (axios, remoteLcdURL) => {
       }
       return req(`GET`, `/auth/accounts/${address}`)()
         .then(res => {
-          return res.value || emptyAccount
+          // HACK, hope for: https://github.com/cosmos/cosmos-sdk/issues/3885
+          let account = res.value || emptyAccount
+          if (res.type === `auth/DelayedVestingAccount`) {
+            account = Object.assign(
+              account.BaseVestingAccount.BaseAccount, res.BaseVestingAccount
```
Comment (fedekunze) [inline app/src/renderer/connectors/api.js]: I'd take a defensive approach here

File: PENDING.md (hunk 1 above)
Comment (fedekunze) [reply PENDING.md]: and not sure we should close the epic issue

**Comment by fedekunze** [comment]
@faboweb how do I test this ?

**Comment by faboweb** [comment]
@fedekunze 

> Simple version where we display the coins as normal coins. This is the lease code change (more like a bugfix). We can think about adding more vesting info later.

**Pull request merged**